administrator/components/com_config/tmpl/application/default.php

```diff
@@ -31,9 +31,9 @@
 <form action="<?php echo Route::_('index.php?option=com_config'); ?>" id="application-form" method="post" name="adminForm" class="d-flex form-validate" data-cancel-task="config.cancel.component">
 	<!-- Begin Sidebar -->
 	<div id="sidebar" class="com-config-sidebar">
-		<button class="btn btn-sm btn-secondary my-2 options-menu d-md-none" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="<?php echo Text::_('TPL_SPRING_TOGGLE_SIDEBAR'); ?>">
+		<button class="btn btn-sm btn-secondary my-2 options-menu d-md-none" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="<?php echo Text::_('TPL_KHONSU_TOGGLE_SIDEBAR'); ?>">
```

Comment:
You should not use a template specific language string in a core non template file. See https://github.com/joomla/joomla-cms/pull/27082 for reference